modin/pandas/dataframe.py

```diff
@@ -2498,6 +2498,10 @@ def __setattr__(self, key, value):
             pass
         elif key in self and key not in dir(self):
             self.__setitem__(key, value)
+            # Note: Return so that we don't keep track of another attribute. Instead
+            # we will register a "ghost" attribute, that will be picked up when
```

Comment:
Sorry if this is pedantic, but what is the `"ghost"` attribute? We catch a missing attribute error in `__getattr__` and then return the column if it's a column name. Is this the `"ghost"` attribute?

Reply:
I went ahead and rewrote this comment